controllers/flowcollector_controller_test.go

```diff
@@ -326,14 +323,6 @@ func flowCollectorControllerSpecs() {
 				Expect(ds.Spec.Template.Spec.Tolerations).
 					To(ContainElement(v1.Toleration{Operator: v1.TolerationOpExists}))
 			})
-
-			By("Expecting the monitoring dashboards configmap to be deleted")
```

Comment:
I didn't realize that, but I guess that's the drawback of not using FLP: we loose the ability to tie dashboards with metrics presence. Hence this test not passing, I guess. Can we think about a simple alternative for that? Maybe it requires some discussion?

Reply:
done in https://github.com/netobserv/network-observability-operator/pull/388/commits/71da325c421290cf515e8ef6aed34b6b60db0577 with other enhancements described below